src/api/app/models/package.rb

```diff
@@ -76,6 +76,7 @@ class Package < ApplicationRecord
 
   after_save :write_to_backend
   after_save :populate_to_sphinx
+  after_save :backfill_bs_request_actions
```

Comment:
```suggestion
  after_create :backfill_bs_request_actions
```

Reply:
We only need to do this if we are creating a new record. There is no way to update `Project.name/Package.name`.

Reply:
Suggestion applied.